sdk/storage/storage-queue/src/QueueClient.ts

```diff
@@ -628,10 +575,12 @@ export class QueueClient extends StorageClient {
   public async create(options: QueueCreateOptions = {}): Promise<QueueCreateResponse> {
     const { span, updatedOptions } = createSpan("QueueClient-create", options);
     try {
-      return await this.queueContext.create({
-        ...updatedOptions,
-        abortSignal: options.abortSignal,
-      });
+      return assertResponse<QueueCreateHeaders, QueueCreateHeaders>(
+        await this.queueContext.create({
+          ...updatedOptions,
+          abortSignal: options.abortSignal,
```

Comment:
is `abortSignal` being yanked out of the bag by `createSpan`? If not, then we could simplify this by passing `updatedOptions` as is.